src/python/tools/manifest_diff.json

```diff
@@ -176,7 +176,6 @@
     "Common.MakeArrayIDataView",
     "Common.MakeArrayIPredictorModel",
     "CountTable.Create",
-    "Dssm.Trigram",
```

Comment:
Does entrypoint compiler need to be re-run after making this change?

Reply:
There's no change to API or entrypoints with this change.